### ISSUE TITLE
gfortran/ATLAS runtime error for 2016

### ISSUE THREAD

**Issue by virtuald** [robotpy/roborio-packages#9]
Not totally unexpected, but I figured I'd try it with the existing libraries:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.5/site-packages/numpy/__init__.py", line 180, in <module>
    from . import add_newdocs
  File "/usr/local/lib/python3.5/site-packages/numpy/add_newdocs.py", line 13, in <module>
    from numpy.lib import add_newdoc
  File "/usr/local/lib/python3.5/site-packages/numpy/lib/__init__.py", line 8, in <module>
    from .type_check import *
  File "/usr/local/lib/python3.5/site-packages/numpy/lib/type_check.py", line 11, in <module>
    import numpy.core.numeric as _nx
  File "/usr/local/lib/python3.5/site-packages/numpy/core/__init__.py", line 14, in <module>
    from . import multiarray
ImportError: /usr/local/lib/libatlas.so: undefined symbol: __aeabi_uidiv
```

There aren't instructions for updating ATLAS/libgfortran, otherwise I'd give it a try. It's probably not that bad though.. right? :p

**Comment by PeterJohnson** [comment]
Updating ATLAS/libgfortran is a real pain.  Is this due to a new version of numpy?

**Comment by virtuald** [comment]
No, I think this is due to the updated RoboRIO image.

**Comment by virtuald** [comment]
I was able to build libgfortran using bitbake and the openembedded toolchain setup from NI. Haven't gotten to ATLAS yet.

**Comment by virtuald** [comment]
I think I have ATLAS built now too (on the roborio itself), I'll upload it and try out numpy tomorrow. I've also got notes on how I built it, so that'll be useful for the next time it needs to be done.

**Comment by PeterJohnson** [comment]
Great!  I just didn't have time this year to tackle it.

**Comment by virtuald** [comment]
Fixed in 810281ccf37f6ea747b7e45606fda10aea8afc96
